types/codemirror/addon/display/autorefresh.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for CodeMirror
```

Comment:
Needs to be `codemirror`

Reply:
Fixed (all files)
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42669/commits/284fb532d33c7f3c6c5b5287117b61595bba630a